float32 JoyLeft_X
float32 JoyLeft_Y
float32 JoyRight_X
float32 JoyRight_Y
float32 LT
float32 RT

 
